docs/source/io_formats/settings.rst

```diff
@@ -321,14 +321,14 @@ or sub-elements:
 
   :method:
 
-    Which resonance elastic scattering method is to be applied: "ares"
-    (accelerated resonance elastic scattering), "dbrc" (Doppler broadening
-    rejection correction), or "wcm" (weight correction method). Descriptions of
-    each of these methods are documented here_.
+    Which resonance elastic scattering method is to be applied: "rvs" (relative
+    velocity sampling), "dbrc" (Doppler broadening rejection correction), or
+    "wcm" (weight correction method). Descriptions of each of these methods are
```

Comment:
Remove reference to wcm here

Reply:
Nice catch -- just removed that and one other reference to wcm I found